test/cppsuite/configs/schema_sweep_stress.txt

```diff
@@ -0,0 +1,34 @@
+# Configuration for schema_sweep.
+duration_seconds=3600,
```

Comment:
nit: Could we get the same results by only running for 5/10 minutes? An hour seems long

Reply:
Yes it should give us enough signal, updated in 2a7693811ef5773257436cd338c36ff4abc89d48